README.md

```diff
@@ -1,4 +1,30 @@
 # ThunderDome
-## Two Goblins Enter One Goblin Leaves ##
+## Two Monsters Enter One Monster Leaves ##
 
 This is a stripped down D&D 5e combat simulator that I am making for practice.
+
+## Build and Run
+
+To build and run this project, first clone it from Github and navigate to the root of the project.  Open a terminal and run the following:
+
+```sh
+# Build
+dotnet build ThunderDome/
```

Comment:
I think it is a good idea to tell folks how to build and run your project in the readme